pytorch/inference/docker/1.10/py3/Dockerfile.e3.graviton

```diff
@@ -152,4 +152,4 @@ RUN curl https://aws-dlc-licenses.s3.amazonaws.com/pytorch-1.10/license.txt -o /
 
 EXPOSE 8080 8081
 ENTRYPOINT ["python", "/usr/local/bin/dockerd-entrypoint.py"]
-CMD ["torchserve", "--start", "--ts-config", "/home/model-server/config.properties", "--model-store", "/home/model-server/"]
+CMD ["torchserve", "--start", "--ts-config", "/home/model-server/config.properties"]
```

Comment:
same here

Reply:
we can have `CMD["serve"]` here, but will need to modify the entrypoint.py to match the open source behaviour